README.md

```diff
@@ -76,44 +82,51 @@ flake8 .
+ Please install and set up the following packages first. Upgrade if you find the package is already installed:
 
-* Download [Python3](https://www.python.org/downloads/). It is advisable to install the python package as an administrator. Click on the 'Add path' checkbox before moving on to the next step of the installation process. Run the 'python3 -V' command to see the version you have installed.
+* Download [Python3](https://www.python.org/downloads/). It is advisable to install the python package as an administrator. Click on the 'Add path' checkbox before moving on to the next step of the installation process. Run this command in your terminal to see the version you have installed.
+  ```sh
+  python3 -V
+  ```
 
 * Download [pip](https://pip.pypa.io/en/latest/installing/) and follow the instructions in the link as an installation guide.
 
+* Download and install [Git](https://git-scm.com/download/win). Clone the project to a fresh directory/folder from the `staging` branch. 
+
 * [PostgreSQL](https://www.postgresql.org/download/windows/) (Ensure the server is running). 
 
-* It is advisable to install Django in a virtual environment and the README uses [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation) to create this environment. You could use any virtualenv package of your choice but for windows install this wrapper with:
+* It is advisable to install Django in a virtual environment. The README uses [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation) to create this environment. You could use any virtualenv package of your choice but for Windows, install this wrapper with:
 
-```sh
-py -m pip install virtualenvwrapper-win
-```
+  ```sh
+  py -m pip install virtualenvwrapper-win
+  ```
 
-* Create a new virtual environment:
+* Then, create a new virtual environment:
 
-```sh
-mkvirtualenv <envname>
-```
+  ```sh
+  mkvirtualenv <envname>
+  ```
 
-```sh
-workon <envname>
-```
+* Activate the virtual environment with: 
+  ```sh
+  workon <envname>
+  ```
 
 * Install requirements in the virtual environment created:
 
-```sh
-py -m pip install Django
-```
+  ```sh
+  py -m pip install Django
+  ```
 
-```sh
-pip install -r requirements.txt
-```
+  ```sh
+  pip install -r requirements.txt
+  ```
 
-* Create a database with PostgresQL, if you installed it earlier. If not, installation instructions can be found [here](https://www.postgresql.org/download/windows/)
+* Create a database with PostgresQL, if you installed it earlier. If not, installation instructions can be found [here](https://www.postgresql.org/download/windows/). Make sure to note Database name, Database Username and Password.
 
-* Create a .env file and copy the content of `.env.example` file to it.
-* Replace
+* Create a .env file and copy the content of `.env.example` file to it. Place this file in the folder named `core`.
```

Comment:
nit: How about editing the last sentence to _place this file in the root directory of the core project_